src/selectors/entities/users.ts

```diff
@@ -354,9 +357,9 @@ export function searchProfiles(state: GlobalState, term: string, skipCurrent = f
     return filteredProfiles;
 }
 
-export function searchProfilesInChannel(state: GlobalState, channelId: $ID<Channel>, term: string, skipCurrent = false): Array<UserProfile> {
+export function searchProfilesInChannel(state: GlobalState, channelId: $ID<Channel>, term: string, skipCurrent = false, skipInactive = false): Array<UserProfile> {
     const doGetProfilesInChannel = makeGetProfilesInChannel();
-    const profiles = filterProfilesMatchingTerm(doGetProfilesInChannel(state, channelId, false), term);
+    const profiles = filterProfilesMatchingTerm(doGetProfilesInChannel(state, channelId, skipInactive), term);
```

Comment:
Not something specific to this PR, but making a new `doGetProfilesInChannel` selector as well as filtering each time this is called gets rid of any memoization benefits from the selector, so it may cause performance issues due to constant re-rendering of components. Depending on where this is used, it could cause be worse, but it'll probably have less effect in the System Console.